bpf/bpf_lb.c

```diff
@@ -194,9 +243,14 @@ int from_netdev(struct __sk_buff *skb)
 	}
 
 	if (IS_ERR(ret))
-		return send_drop_notify_error(skb, ret, TC_ACT_SHOT);
+		goto drop;
+
+#ifdef ENCAP_IFINDEX
+	cilium_trace_capture(skb, DBG_CAPTURE_DELIVERY, ENCAP_IFINDEX);
+	return ret;
+#else /* ENCAP_IFINDEX */
 
-#ifdef LB_REDIRECT
+#if defined LB_REDIRECT
```

Comment:
why the churn from `ifdef` to `if defined` do you plan for LB_REDIRECT to be compound conditional